CHANGELOG.md

```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
 `optionsTemplateURL` will display the template in the side navigation menu.
 This may require minor layout/appearance adjustments in the template. (#684)
 
++ To upgrade:
```

Comment:
:art: `23:3  warning  Incorrect list-item indent: add 2 spaces  list-item-indent  remark-lint`

I suspect this is being confused by lines 24-26, the parser isn't seeing enough spaces to determine these are sub-list items.